1_fetch/src/download_sb_data.r

```diff
@@ -5,13 +5,12 @@
   library(sbtools)
   library(whisker)
   
-  fetch_data <- function() {
-    project_output_dir <- '1_fetch/out'
+  fetch_data <- function(project_output_dir) {
     #create directory
-    dir.create(project_output_dir)
+    dir.create(project_output_dir, showWarnings = FALSE)
     
     
     # Get the data from ScienceBase
-    mendota_file <- file.path(project_output_dir, 'model_RMSEs.csv')
+    mendota_file <- file.path(project_output_dir)
```

Comment:
Now that you're passing in the full output file path, there's no need to create this intermediate `mendota_file` variable. Instead you can just use the name of your filepath argument in the call to `item_file_download`